src/sentry/nodestore/bigtable/backend.py

```diff
@@ -231,7 +238,11 @@ def delete(self, id):
 
         row = self.connection.row(id)
         row.delete()
-        row.commit()
+
+        status = row.commit()
+        if status.code != 0:
+            raise BigtableError(status.code, status.message)
```

Comment:
any way we can get that impld for delete_multi too?

Reply:
Oops, yeah. That was an oversight on my part, will include there as well. Thanks for the reminder.

Reply:
Added in e9d96fc4612262b5392c2a2762f6646ea372dde3, thanks again for calling that out.